assets/sass/modules/navigation/_mobile-navigation.scss

```diff
@@ -26,11 +26,11 @@
 	} // li
 
 	a {
-		color: $color-white;
+		color: $color-mobile-menu-link;
 		font-size: rem(14);
 
 		&:focus {
-			outline: rem(2) solid $color-white;
+			outline: rem(2) solid $color-mobile-menu-link;
```

Comment:
We're setting `outline` with `rem()` here and then `px` later on in the file. We don't have a specification in our Units of Measurement graphic... do we need an update to a new version after over a year at v0.3.0?! 😮